packages/palette/src/themes/types.ts

```diff
@@ -1,14 +1,9 @@
-import {
-  Breakpoint as BreakpointV2,
-  Color as ColorV2,
-  SpacingUnit as SpacingUnitV2,
-} from "@artsy/palette-tokens/dist/themes/v2"
 import {
   Breakpoint as BreakpointV3,
   Color as ColorV3,
   SpacingUnit as SpacingUnitV3,
 } from "@artsy/palette-tokens/dist/themes/v3"
 
-export type Color = ColorV2 | ColorV3
-export type SpacingUnit = SpacingUnitV2 | SpacingUnitV3
-export type Breakpoint = BreakpointV2 | BreakpointV3
+export type Color = ColorV3
+export type SpacingUnit = SpacingUnitV3 | `${number}px`
```

Comment:
If this was to actually constrain the types, then we would have only be able to use the union of v2 and v3 before, which was not the case.

Reply:
Updated, had to pass into the generic slot